site/_scss/home.scss

```diff
@@ -0,0 +1,36 @@
+//HOME
+
+// Globals
+@import 'globals/functions';
+@import 'globals/colors';
+@import 'globals/vars';
+@import 'globals/mixins';
+@import 'globals/gorko';
+@import 'globals/extends';
+@import 'globals/common-blocks';
+
+@import 'fonts';
+@import 'theme';
+@import 'reset';
+@import 'global';
+@import 'typography';
```

Comment:
Sorry I am now going to ask for quite an operation here: we'll most likely use this block across all templates, no? So I would move _fonts.scss, _theme.scss, _reset.scss, and _typography.scss into the globals directory to and then add this block to _global.scss and then only import this here. I think this would make things clearer.